bank2ynab.py

```diff
@@ -622,14 +646,33 @@ def _fix_decimal_point(self, row):
 
         return row
 
+    def _clean_monetary_values(self, row):
```

Comment:
Ooh, this is a potentially dangerous function. You have it in the right place in the code though - in theory, any currency using apostrophes as decimal points would have been fixed by this point.

Reply:
Yeah. I made sure it was after the conversion of any `,` to `.` By this point it should be safe. (That's what unittests are for!)